### DISCUSSION TITLE
Show-InstallationWelcome functionality

### DISCUSSION THREAD

**Discussion by rpolito2717** [PSAppDeployToolkit/PSAppDeployToolkit#845] [Q&A] [answered]
I have been using the AppDeploy Toolkit for some time but I haven't had a need to use Show-InstallationWelcome with the various defer switches.  I understand the parameters but I am looking more for an understanding of what happens when something like this is used.
Show-InstallationWelcome -CloseApps 'iexplore' -AllowDefer -DeferTimes 3
I know it will prompt to close IE, allow the install to be deferred up to 3 times, but what makes it prompt or trigger the installation/script to run again?  Does a scheduled take get created?  Looking for the mechanism for what triggers the execute again.

**Comment by Klumpatsch-trex** [comment, marked as answer]
Hello there, if youre still waiting:
To my mind PSAppDeployToolkit is suited for deployment via Softwaredeployment like SCCM.
If you deploy via SCCM, Softwarecenter will detect deployment as "failed" if you choose to differ and try again in some time.
(Differ Counter is stored in Registry)
DeployToolkit itself wont retry the execution.
